test/toolchain/test_repo/.plzconfig

```diff
@@ -8,3 +8,6 @@ BuildFileName = BUILD_FILE
 BuildFileName = BUILD
 PreloadSubincludes = ///go//build_defs:go
 
+[remote]
+url =
```

Comment:
this makes local dev stuff here a bit easier; it shouldn't affect the test otherwise (but it never actually works because of how we set up the toolchain)